From d38afdb5bfc51bc3fa4db4381174277f362650bc Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Sun, 18 Apr 2021 18:48:28 +0200
Subject: [PATCH] python3-python: remove it

This package, we need for Turris OS 5.x, which was based on top of
OpenWrt 19.07, because there were two versions of Python. In upcoming
version of OpenWrt 21.02, there is just one Python version and
python3-base provides this symlink.

Take a look here:
https://github.com/openwrt/packages/blob/6631cfaa61ff75d97ef1a41c6ec031198103c7df/lang/python/python3/Makefile#L277

So we no longer need this package
---
 lang/python3-python/Makefile | 35 -----------------------------------
 1 file changed, 35 deletions(-)
 delete mode 100644 lang/python3-python/Makefile

diff --git a/lang/python3-python/Makefile b/lang/python3-python/Makefile
deleted file mode 100644
index 9731619e1..000000000
--- a/lang/python3-python/Makefile
+++ /dev/null
@@ -1,35 +0,0 @@
-#
-## Copyright (C) 2020 CZ.NIC z.s.p.o. (http://www.nic.cz/)
-#
-## This is free software, licensed under the GNU General Public License v2.
-# See /LICENSE for more information.
-# #
-#
-include $(TOPDIR)/rules.mk
-
-PKG_NAME:=python3-python
-PKG_VERSION:=1.0.0
-PKG_RELEASE:=1
-PKG_MAINTAINER:=CZ.NIC <packaging@turris.cz>
-PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)
-
-include $(INCLUDE_DIR)/package.mk
-
-define Package/python3-python
-  TITLE:=Python3 python link
-  DEPENDS:=+python3-base
-  CONFLICTS:=python-base
-endef
-
-define Package/python3-python/description
-  This package provides python link to python3. 
-endef
-
-define Package/python3-python/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	ln -s /usr/bin/python3 $(1)/usr/bin/python
-endef
-
-Build/Compile:=:
-
-$(eval $(call BuildPackage,python3-python))
-- 
2.25.1

